### PR TITLE
Payment fixes

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6581

*Description of changes:*

* Some more action refactors.
* Fixes rollback request.
* Fixes Put Requests after initial create.
* Fixes the payment completion flow.

* Next steps: EDIT & Priority Request updates on EXISTING REQUESTS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/common/request-details.vue [hunk 1]
```diff
@@ -2,18 +2,20 @@
...
+          <li v-if="name && !nameChoices.name2">{{name}}</li>
+          <template v-else>
+            <li v-if="nameChoices.name1">
+              <span>1. </span>{{nameChoices.name1}}
+            </li>
+            <li v-if="nameChoices.name2">
+              <span>2. </span>{{nameChoices.name2}}
+            </li>
+            <li v-if="nameChoices.name3">
+              <span>3. </span>{{nameChoices.name2}}
```
Comment (severinbeauvais) [inline src/components/common/request-details.vue:16]: Should be name3.

Comment (severinbeauvais) [reply src/components/common/request-details.vue:16]: Make sure you test this. It looks OK but I'm a bit cross-eyed atm :>

File: src/interfaces/store-interfaces/action-interface.ts [hunk 2]
```diff
@@ -6,5 +6,5 @@ export interface ActionIF {
 }
 
 export interface ActionBindingIF {
-  (x: any | null, y?: any | null): any
+  (x: any | null, y?: any | null, z?: any | null): any
```
Comment (severinbeauvais) [inline src/interfaces/store-interfaces/action-interface.ts:9]: Really? lol

**Review by severinbeauvais — APPROVED** [review]
Found a typo but otherwise LGTM.

Nice cleanup 👍

File: src/interfaces/store-interfaces/action-interface.ts (hunk 2 above)
Comment (cameron-freshworks) [reply src/interfaces/store-interfaces/action-interface.ts:9]: Oops sorry, i was testing something with actions and the way it handles additional parameters. 
Which it doesn't! So i will remove this :)

File: src/components/common/request-details.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/common/request-details.vue:16]: oo thanks!

Comment (cameron-freshworks) [reply src/components/common/request-details.vue:16]: Will fix

File: src/interfaces/store-interfaces/action-interface.ts (hunk 2 above)
Comment (cameron-freshworks) [reply src/interfaces/store-interfaces/action-interface.ts]: Fixed!

File: src/components/common/request-details.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/common/request-details.vue]: Fixed!

**Pull request merged**